protocol/avdl/favorite.avdl

```diff
@@ -15,6 +15,12 @@ protocol favorite {
     boolean created;          // this folder was just created by this user
   }
 
+  record FavoritesResult {
```

Comment:
could you add a comment on this struct?  I'm asking you about it in slack, but I'm sure I'll forget and it isn't quite clear what each field contains.